src/pages/events/index.tsx

```diff
@@ -1,19 +1,24 @@
 import React, { useEffect, useState } from "react"
 import EventCard from "@/components/events/event-card"
-import { useSupabaseClient } from "@supabase/auth-helpers-react"
+import {useSupabaseClient, useUser} from "@supabase/auth-helpers-react"
 import { Events, EventWithAttendance } from "@/types/database-types"
 import { getEventsWithAttendees } from "@/services/event.service"
 import { Database } from "@/types/database"
-import { ExclamationTriangleIcon } from "@heroicons/react/24/outline"
+import {ExclamationTriangleIcon, XMarkIcon} from "@heroicons/react/24/outline"
 import Spinner from "@/components/utils/spinner"
+import { PlusIcon } from "@heroicons/react/24/solid";
+import { useRouter } from "next/router";
```

Comment:
* On line 3 & 7 it is missing spaces around the imported elements
* XMarkIcon is unused
* Line 9 + 10 have unwanted semicolons. 

I noticed that Kevin also mentioned several of these type of things earlier. Can you double check if you have Prettier properly installed and configured? (https://prettier.io/docs/en/editors.html) And make sure that it runs on save
![image](https://github.com/ASVGay/the-rhapsodies/assets/54884540/1da8fadd-4bf5-493b-a093-080b7364e7de)

It will otherwise only generate more line changes for us in the future since they will have to prettify your code. It should also automatically run every time you commit. Try running the prepare script in package.json 
![image](https://github.com/ASVGay/the-rhapsodies/assets/54884540/4d0919a4-8172-4743-8936-944205506676)